src/main/java/com/jagrosh/giveawaybot/util/OtherUtil.java

```diff
@@ -76,4 +81,46 @@ public static int parseWinners(String winstr)
             return -1;
         return Integer.parseInt(winstr.substring(0, winstr.length()-1));
     }
+
+    public static boolean validateEmoji(Message message, String rawEmoji)
+    {
+        if (rawEmoji == null || rawEmoji.isEmpty())
+            return true;
+        try
+        {
+
+            message.addReaction(rawEmoji).complete();
```

Comment:
I'm not a fan of having this non-async method in a util, but I guess I need to think more about how we should actually handle this...

In general, I think our flow should look more like:
1. User provides the inputs they desire for a giveaway
2. We attempt to create the giveaway
3. We report back to the user if it succeeded, or if not, why it failed.  

This flow will be especially important once we switch to using slash commands, as we won't be able to pre-check as many things. It might be worth getting our internal logic to slash-command level to facilitate the switch.

Reply:
```java
public enum SomeEnum
{
    CANT_SEND, DATABASE, SUCCESS, TOO_MANY;
    private SomeEnum(boolean successful, String details)
    {}
}

public SomeEnum startGiveaway(...)
{
    return SomeEnum.CANT_SEND;
}
```
I'm suggesting an enum or similiar here for having a more detailed feedback.